transport-classes-io_uring/src/main/java/io/netty/channel/uring/AbstractIoUringServerChannel.java

```diff
@@ -143,20 +144,38 @@ protected int scheduleRead0(boolean first, boolean socketIsEmpty) {
             } else {
                 ioPrio = 0;
             }
+
+            if (IoUring.isIOUringAcceptMultishotSupported()) {
+                // Let's use multi-shot accept to reduce overhead.
+                ioPrio |= Native.IORING_ACCEPT_MULTISHOT;
```

Comment:
"Note that for the multishot variants, setting addr and addrlen may not make a lot of sense, as the same value would be used for every accepted connection. This means that the data written to addr may be overwritten by a new connection before the application has had time to process a past connection"

https://man7.org/linux/man-pages/man3/io_uring_prep_multishot_accept_direct.3.html

so we should call `getpeername` in readComplete0 to get real peer address

Reply:
Good point!

Reply:
Fixed @dreamlike-ocean PTAL again